### PR TITLE
office-js-preview add DialogParentMessageReceivedEventArgs

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Please fill in this template.

- [ x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:
If changing an existing definition:
- [x ] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.microsoft.com/en-us/office/dev/add-ins/develop/parent-to-dialog
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -5104,6 +5104,25 @@ declare namespace Office {
          */
         type: EventType;
     }
+    /**
+     * Provides information about the message from the parent page that raised the DialogParentMessageReceived event.
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts]: Style suggestions:
```suggestion
     * Provides information about the message from the parent page that raised the "DialogParentMessageReceived" event.
```

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -5104,6 +5104,25 @@ declare namespace Office {
...
+     * 
+     * @beta
+     * 
+     * To add an event handler for the DialogParentMessageReceived event, use the addHandlerAsync method of the
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts]: Style suggestions
```suggestion
     * To add an event handler for the "DialogParentMessageReceived" event, use the `addHandlerAsync` method of the 
```

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -5104,6 +5104,25 @@ declare namespace Office {
...
+     * 
+     * @beta
+     * 
+     * To add an event handler for the DialogParentMessageReceived event, use the addHandlerAsync method of the 
+     * {@link Office.UI} object.
+     * 
+     */
+    interface DialogParentMessageReceivedEventArgs {
+        /**
+         * Gets the content of the message sent from the parent page, which can be any data that can be stringified.
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts]: The double "can" is little repetitive.
```suggestion
         * Gets the content of the message sent from the parent page, which can be any stringified data.
```

**Review by AlexJerabek — APPROVED** [review]
Looks good. Just a couple suggestions around style. @ElizabethSamuel-MSFT can confirm, but I think we're trying to use quotation marks around enum values and code ticks around method/property names.

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply types/office-js-preview/index.d.ts]: If this is the code name of the event, I would recommend backticks.

**Pull request merged**